core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -728,6 +726,11 @@ protected function doReplace($strBuffer, $blnCache)
 					else
 					{
 						// Do not use StringUtil::specialchars() here (see #4687)
+						if (!\in_array($elements[1], array('title', 'parentTitle', 'mainTitle', 'rootTitle', 'pageTitle', 'parentPageTitle', 'mainPageTitle', 'rootPageTitle'), true))
+						{
+							$flags[] = 'attr';
+						}
```

Comment:
@ausi Can you please check if we have to add this inside the `if` condition above as well?

Reply:
No, we don’t have to, because encoding is already handled correctly there.